### PR TITLE
[azure-iot-sdk-c] LTS_08_2023

### PR DESCRIPTION
**Pull request by RLeclair**
[azure-iot-sdk-c] LTS_08_2023

### CODE REVIEW

File: ports/azure-iot-sdk-c/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-iot-sdk-c",
-  "version-date": "2023-08-06",
+  "version-date": "2023-08-03",
```
Comment (dg0yt) [inline ports/azure-iot-sdk-c/vcpkg.json:3]: Now we are moving backwards.

Comment (RLeclair) [reply ports/azure-iot-sdk-c/vcpkg.json]: Pushed control date forward to `2023-08-06`

Comment (RLeclair) [reply ports/azure-iot-sdk-c/vcpkg.json]: Pushed to `2023-08-07`

**Review by vaavva — APPROVED** [review]

**Comment by RLeclair** [comment]
Hi @dg0yt, anything else needed to merge this PR?

File: ports/azure-macro-utils-c/vcpkg.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-macro-utils-c",
-  "version-date": "2022-01-21",
+  "version-date": "2023-08-07",
```
Comment (FrankXie05) [inline ports/azure-macro-utils-c/vcpkg.json:3]: Please modify the corresponding `portfile.cmake` file.

File: ports/umock-c/vcpkg.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "umock-c",
-  "version-date": "2022-01-21",
+  "version-date": "2023-08-07",
```
Comment (FrankXie05) [inline ports/umock-c/vcpkg.json:3]: https://github.com/microsoft/vcpkg/blob/ad46340bfce415333d6a2139592c22a499fb0df0/ports/umock-c/portfile.cmake#L6

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

**Comment by FrankXie05** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". 
          That way, I can be aware that you've responded since you can't modify the tags.

File: ports/umock-c/vcpkg.json (hunk 3 above)
Comment (dg0yt) [reply ports/umock-c/vcpkg.json:3]: Can we be explicit?
Either don't update the `version-date`, or actually update the port.

File: ports/azure-macro-utils-c/vcpkg.json (hunk 2 above)
Comment (dg0yt) [reply ports/azure-macro-utils-c/vcpkg.json:3]: i.e. either don't update the version-date, or actually update the port.

**Review by dg0yt — CHANGES_REQUESTED** [review]

File: ports/azure-macro-utils-c/vcpkg.json (hunk 2 above)
Comment (RLeclair) [reply ports/azure-macro-utils-c/vcpkg.json]: Removed update to `version-date`

File: versions/a-/azure-c-shared-utility.json [hunk 4]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "ae2cd1418fa9f6d28a8e515d972c1eb982ed7df8",
+      "version-date": "2023-08-07",
+      "port-version": 1
```
Comment (FrankXie05) [inline versions/a-/azure-c-shared-utility.json:6]:
```suggestion
      "port-version": 0
```
Need to delete the `port-version` field in the `vcpkg.json`, and use the command `./vcpkg x-add-version`

**Review by FrankXie05 — APPROVED** [review]

**Pull request merged**